pkg/cmd/inletsoperator_app.go

```diff
@@ -193,3 +171,26 @@ func getOverridesWithPlatform(command *cobra.Command) map[string]string {
 
 	return overrides
 }
+
+const inletsOperatorInfoMsg = `# The default configuration is for DigitalOcean and your secret is
+# stored as "inlets-access-key" in the "default" namespace.
+
+# To get your first Public IP run the following:
+kubectl run nginx-1 --image=nginx --port=80 --restart=Always
+kubectl expose deployment nginx-1 --port=80 --type=LoadBalancer
+
+# Find your IP in the "EXTERNAL-IP" field, watch for "<pending>" to 
+# change to an IP
+
+kubectl get svc -w
+
+# When you're done, remove the tunnel by deleting the service
+kubectl delete svc/nginx-1
+
+# Find out more at:
+# https://github.com/inlets/inlets-operator`
+
+const inletsOperatorPostInstallMsg = `=======================================================================
+= inlets-operator has been installed.                                  =
```

Comment:
Did we get an extra space in the line or does this still align OK?

Reply:
There are two newlines in between the three sections -

https://github.com/alexellis/k3sup/pull/137/files/77257c4a8e56565bfd18906f94127c6045818e98#diff-797c244a74f9a274b0072f00ce0e0995L145-L147

https://github.com/alexellis/k3sup/pull/137/files/77257c4a8e56565bfd18906f94127c6045818e98#diff-797c244a74f9a274b0072f00ce0e0995L163-L165

For the first one I used one newline inside the backtick string. I'll change it to use `\n\n` too, similar to the next one. It will be more clear and consistent